test/test_ensemble_builder/test_ensemble.py

```diff
@@ -463,3 +466,62 @@ def testPredict(self):
 
         with self.assertRaises(ValueError):
             ensemble.predict(per_model_pred)
+
+
+class SingleModelEnsembleTest(unittest.TestCase):
+    def test_get_identifiers_from_run_history(self):
+        run_history = RunHistory()
+        run_history._add(
+            RunKey(
+                config_id=3,
+                instance_id='{"task_id": "breast_cancer"}',
+                seed=1,
+                budget=3.0
+            ),
+            RunValue(
+                cost=0.11347517730496459,
+                time=0.21858787536621094,
+                status=None,
+                additional_info={
+                    'duration': 0.20323538780212402,
+                    'num_run': 3,
+                    'configuration_origin': 'Random Search'}
+            ),
+            status=None,
+            origin=None,
+        )
+        run_history._add(
+            RunKey(
+                config_id=6,
+                instance_id='{"task_id": "breast_cancer"}',
+                seed=1,
+                budget=6.0
+            ),
+            RunValue(
+                cost=2*0.11347517730496459,
+                time=2*0.21858787536621094,
+                status=None,
+                additional_info={
+                    'duration': 0.20323538780212402,
+                    'num_run': 6,
+                    'configuration_origin': 'Random Search'}
+            ),
+            status=None,
+            origin=None,
+        )
+        ensemble = SingleModelEnsemble(
+             ensemble_size=5,
+             task_type=None,
+             metric=accuracy,
```

Comment:
Could you please add a test with a loss function (i.e. logloss), too?

Reply:
Added a check for this. BTW, thanks for flagging this, I was able to find a bug in the code due to this, which has been corrected. Also added extra checks to capture this.